tests/integration/update_cluster/minimal-cloudformation/cloudformation.json.extracted.yaml

```diff
@@ -26,6 +26,8 @@ Resources.AWSAutoScalingLaunchConfigurationmasterustest1amastersminimalexampleco
 
 
 
+
```

Comment:
Starting to wonder if we should automatically replace "\n\n" -> "\n" :-) 

(not in this PR though, unless you really want to!)

Reply:
Agreed! I was trying to avoid scope creep here so I didn't bother to change it (also I'm lazy 😆), happy to follow up in another PR :)